frappe/utils/__init__.py

```diff
@@ -19,7 +19,7 @@
 from typing import Generator, Iterable
 
 from six import string_types, text_type
-from six.moves.urllib.parse import quote
+from six.moves.urllib.parse import quote, urlparse
```

Comment:
I think if we're moving towards Py3 only, we should remove references to the `six` module

Reply:
six.moves.urllib.parse will be changed to urllib.parse

what about `string_types` and `text_type`? just replace all instances with `str`? https://six.readthedocs.io/#six.string_types

Reply:
Yeah, they're aliases anyway, `str` is fine